main/tests.py

```diff
@@ -76,4 +78,59 @@ def test_dog_owner_homepage_is_visible_for_dog_owner(self, client, create_dog_ow
         client.force_login(user=create_dog_owner_user.user)
         response = client.get("/homepage/")
         assert response.status_code == 200
-        assert list(response.context['daycares']) == list(DayCare.objects.all())
+        assert set(response.context['day_care_queryset']) == set(DayCare.objects.all())
```

Comment:
If this test is ensuring that the page will have a list of all the day cares, maybe it should be mentioned in its name. or in a different test.

Reply:
NP .

Reply:
Fixed by changing the name to " test_dog_owner_homepage_present_all_daycares"